kubernetes/celery-beat/deployment.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: celery-beat-deployment
+  labels:
+    app: celery-beat
+spec:
+  replicas: {{.Values.celeryBeat.replicaCount}}
+  strategy:
+    type: Recreate
```

Comment:
[Importance 5/10]

Recreate (in opposition to RollingUpdate) destroy all then restart all.
This way you effectivly cut the service for a few seconds. Is it expected ?

Reply:
Commented why in the file as discussed, thanks.